src/meta/src/hummock/compaction/picker/base_level_compaction_picker.rs

```diff
@@ -140,15 +140,8 @@ impl LevelCompactionPicker {
                 continue;
             }
 
-            if (!target_level_ssts.is_empty() || input.sstable_infos.len() > 1)
-                && input.sstable_infos.len()
-                    < self.config.level0_sub_level_compact_level_count as usize
-                && input.total_file_size < target_level_size
-                && input.total_file_count < self.config.level0_max_compact_file_number as usize
-            {
-                // not trivial move
-                skip_by_write_amp = true;
-                continue;
+            if input.total_file_size < target_level_size {
```

Comment:
if we remove this condition, it means that we may trigger a task where the number of participating levels < the level0_sub_level_compact_level_count, is this as expected?

Since the min_depth of base level compaction is 1, can we tolerate triggering a compaction task with only one  sub_level (128m) participation?